.semaphore/docker-hub.yml

```diff
@@ -14,10 +14,10 @@ blocks:
           - checkout --use-cache
           - export DOCKER_IMAGE_TAG=${SEMAPHORE_GIT_SHA:0:7}
           - cache restore ${DOCKER_IMAGE_TAG}
-          - make docker/load
-          - make docker/login
+          - make load
```

Comment:
Hm, not sure if this is the right approach. `make load` is not expressive and doesn't really tell you about the target. 

A better approach would be to namespace per domain, e.g.

```
docker/load
docker/load
test/unit
test/pre-commit
....
```

what do you think about that approach ?

Reply:
yes, we have issues in running stuff inside docker and running docker.. so what would you suggest..  don't care where it runs?

Reply:
changed back